cirq-aqt/cirq_aqt/aqt_sampler_test.py

```diff
@@ -105,8 +105,8 @@ def test_aqt_sampler_error_handling():
             max_angle = np.pi
             repetitions = 10
             sampler = AQTSampler(remote_host="http://localhost:5000", access_token='testkey')
-            device, qubits = get_aqt_device(1)
-            circuit = cirq.Circuit(cirq.X(qubits[0]) ** theta, device=device)
+            _, qubits = get_aqt_device(1)
```

Comment:
Is it worth making get_aqt_qubits instead?  Is this called from any other place that actually uses the device?

Reply:
I left it in for minimal impact on the code, it may pop up when it comes time to revisit device validation tests more closely.